Setup_EnvironmentGUI.py

```diff
@@ -0,0 +1,97 @@
+import pygame
+import sys
+
+import Control_EnvironmentGUI as Control_Environment
+import Setup_settingsGUI
+
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+WINDOW_HEIGHT = 770
+WINDOW_WIDTH = 1500
+SCREEN = None
+CLOCK = None
+
+
+class SetupEnvironment:
+    """
+    This will set up the actual environment for the simulation to be displayed in
+    """
+    def __init__(self):
+        global SCREEN, CLOCK
+        pygame.init()
+        SCREEN = pygame.display.set_mode((WINDOW_WIDTH, WINDOW_HEIGHT))
+        CLOCK = pygame.time.Clock()
+        SCREEN.fill(WHITE)
+
+        # The three buttons that are on the environment screen
+        self.settings_button = Button(SCREEN, (1424, 738), "Settings", (255, 255, 0))
+        self.pause_button = Button(SCREEN, (1364, 738), "Pause", (255, 0, 0))
+        self.play_button = Button(SCREEN, (1320, 738), "Play", (0, 255, 0))
```

Comment:
the buttons seem to not be wide enough to fit the text on my screen. It could be a scaling problem? They also overlap the grid. To make sure we can add UI elements without obscuring the board, they should probably be on their own bar which sits below the display of the grid.